File: Service/Billing/RMA.Service.Billing.Contracts/Entities/TermArrangementProductOption.cs
namespace RMA.Service.Billing.Contracts.Entities
{
    public class TermArrangementProductOption
    {
        public int TermArrangementProductOptionId { get; set; } // TermArrangementProducOptionId (Primary key)
        public int ProductOptionId { get; set; } // ProductOptionId
        public int TermArrangementId { get; set; } // TermArrangementId
        public decimal ContractAmount { get; set; } // ContractAmount
        public string ProductOptionName { get; set; }
        public string FinPayenumber { get; set; }
        public int RoleplayerId { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/fincare/src/app/billing-manager/models/term-arrangement-productoption.ts
import { ComplianceResult } from "projects/clientcare/src/app/policy-manager/shared/entities/compliance-result";

export class TermArrangementProductOption{
    productOptionId:number;
    contractAmount:number;
    productOptionName: string;
    finPayenumber: string;
    roleplayerId: number;
    policyId:  number;
    complianceResult: ComplianceResult;

    static copy(termArrangementProductOption: TermArrangementProductOption) :TermArrangementProductOption
    {
       let options =  new TermArrangementProductOption();
       options.productOptionId = termArrangementProductOption.productOptionId;
       options.contractAmount = termArrangementProductOption.contractAmount;
       options.finPayenumber = termArrangementProductOption.finPayenumber;
       options.roleplayerId = termArrangementProductOption.roleplayerId;
       return options;
    }
}
